internal/chart/v3/lint/lint_test.go

```diff
@@ -41,6 +39,7 @@ const malformedTemplate = "rules/testdata/malformed-template"
 const invalidChartFileDir = "rules/testdata/invalidchartfile"
 
 func TestBadChartV3(t *testing.T) {
+	var values map[string]interface{}
```

Comment:
```suggestion
	var values map[string]any{}
```

would be better IMHO, while we are changing this